Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -148,6 +152,7 @@ if(APPLE)
         MACOSX_BUNDLE_BUNDLE_VERSION "0.1"
         MACOSX_BUNDLE_SHORT_VERSION_STRING "0.1"
         RESOURCE "${plClient_XCODE_RESOURCES}"
+	"$<$<VERSION_GREATER_EQUAL:${XCODE_VERSION},26.0>:XCODE_ATTRIBUTE_ASSETCATALOG_OTHER_FLAGS --enable-icon-stack-fallback-generation=disabled>"
```

Comment:
Seems like this might be indented with a tab instead of spaces

Reply:
I think this needs to not be wrapped in quotes, because it's getting interpreted as a string rather than an attribute and value

Reply:
Thanks. I changed it to:
`$<$<VERSION_GREATER_EQUAL:${XCODE_VERSION},26.0>:XCODE_ATTRIBUTE_ASSETCATALOG_OTHER_FLAGS --enable-icon-stack-fallback-generation=disabled>`

That worked but not it's not applying the attribute in my local Xcode 26 install.